ReflectometryServer/ChannelAccess/pv_manager.py

```diff
@@ -306,6 +310,11 @@ def _add_parameter_pvs(self, parameter):
             self._add_pv_with_fields(prepended_alias + DEFINE_POSITION_AS, param_name, align_fields, description,
                                      PvSort.DEFINE_POS_AS)
 
+        # Engineering Unit
+        egu_fields = STANDARD_2048_CHAR_WF_FIELDS.copy()
+        egu_fields["value"] = parameter.engineering_unit
```

Comment:
fields["unit"] = parameter.engineering_unit will make units appear in OPI (I think it was this)